src/map/status.h

```diff
@@ -733,6 +733,7 @@ typedef enum sc_type {
 	SC_STRANGELIGHTS,
 	SC_DECORATION_OF_MUSIC,
 
+	SC_FIRE_EXPANSION_TEAR_GAS_SOB,
```

Comment:
Inserting this here changes the ID of all the SCs that follow (and the comments don't match the generated IDs anymore). It would be better to put it at the end, so that any SCs saved to the database won't turn into the wrong effect on the next restart after applying this patch